docs/modules/ROOT/pages/explanations/decisions/redis_galera.adoc

```diff
@@ -0,0 +1,64 @@
+= Redis and MariaDB/Galera Provisioning
```

Comment:
I'd add a page per service and not mix them in the same decision page. And is it only about provisioning?